Userland/Applications/Browser/Tab.cpp

```diff
@@ -79,6 +79,42 @@ void Tab::view_source(const URL& url, const String& source)
     window->show();
 }
 
+void Tab::update_status(Optional<String> override = {}, i32 count_waiting = 0)
+{
+    if (override.has_value()) {
+        m_statusbar->set_text(*override);
+        return;
+    }
+
+    // Page load event has not fired yet
+    if (!m_loaded) {
```

Comment:
In general comments like these should be removed. It's not particularly valuable to just say what the next line of code is doing, we know that from the code itself, so it just adds noise.